hadoop-ozone/dist/src/main/smoketest/upgrade/validate.robot

```diff
@@ -28,3 +29,25 @@ Read data from previously created key
     ${random} =         Generate Random String  5  [NUMBERS]
     ${output} =         Execute          ozone sh key get /${PREFIX}-volume/${PREFIX}-bucket/${PREFIX}-key /tmp/key-${random}
                         Should not contain  ${output}       Failed
+
+Setup credentials for S3
+# TODO: Run "Setup secure v4 headers" instead when security is enabled
+    Run Keyword         Setup dummy credentials for S3
+
+Read data from previously created key using S3 API
+    ${result} =         Execute AWSS3APICli and checkrc    get-object --bucket ${PREFIX}-bucket --key key1 /tmp/key1.result    0
+    # Note: "Compare files" doesn't work on NOTICE.txt as it is updated in new Ozone versions.
```

Comment:
How about the license file? Should be more stable. We could update the generate/validate for normal keys to use this as well.

Reply:
Good point. I will use LICENSE.txt for now. Once we merge master we could use the new addition: https://github.com/apache/ozone/pull/3050/files#diff-472262337660b3a36c07cf07985181b7b4991652e06e5f9732ab770f79d66dc8R26-R28

Reply:
Whelp. LICENSE.txt isn't guaranteed to work either: https://github.com/apache/ozone/commits/master/LICENSE.txt

I'll just writing a string to compare.